extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -0,0 +1,366 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isExperimentOn} from '../../../src/experiments';
+import {Services} from '../../../src/services';
+import {dev, user} from '../../../src/log';
+import {IntersectionObserverPolyfill} from '../../../src/intersection-observer-polyfill'; // eslint-disable-line max-len
+
+/**
+ * - visibilePercentageMin: The percentage of pixels that need to be on screen
+ *   for the creative to be considered "visible".
+ * - continuousTimeMin: The amount of continuous time, in milliseconds, that
+ *   the creative must be on screen for in oreder to be considered "visible".
```

Comment:
Oops, I didn't mean the order of the object, I meant the word `order` was spelled incorrectly.  LOL